text/0006-architecture-review-process.md

```diff
@@ -40,15 +40,16 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - how the proposal handles these scenarios
       - what review and discussion of the proposed architecture has already happened (and with whom)
     * A competitive analysis suggesting alternatives, costs, and opportunities
+    * Input by representatives from the affected groups. The team is expected to listen to this input and update the review packet based on the input, but it is likely that not all the questions or comments can be answered before the review.
 
-3. The reviewer creates a list of questions to be discussed during the review. These questions should be:
+3. The reviewer creates a list of questions to be discussed during the review, possibly from the comments made in step 2. These questions should be:
     * Open-ended
```

Comment:
You aren't touching this line, but it occurs to me that I'm uncertain why this matters. Is it that open-ended questions are more likely to foster productive analysis and discussion? I wonder if there's more we can say here about the kinds of questions that are helpful for reviews.

Reply:
Yes, that was the point of stipulating open-ended, and we should specify the end not the means. 'Designed to foster productive analysis and discussion' works for me in its place.

Reply:
Fixed in 782db53743233bc3ad8ca5691aee9ab1a73fc49b